x-pack/plugins/apm/public/components/app/ServiceMap/Popover/index.tsx

```diff
@@ -71,13 +71,15 @@ export function Popover({ focusedServiceName }: PopoverProps) {
       cy.on('select', 'node', selectHandler);
       cy.on('unselect', 'node', deselect);
       cy.on('data viewport', deselect);
+      cy.on('drag', 'node', deselect);
```

Comment:
Will dragging cause the popover to show for a brief period of time? Or will it not be displayed at all?

Reply:
Node dragging doesn't trigger the `select` event, so it won't display the popover at all.